warp10/src/main/sh/warp10.sh

```diff
@@ -512,8 +528,11 @@ repair)
 compact)
   compact "$@"
   ;;
+leveldbinit)
+  leveldbinit "$@"
+  ;;
 *)
-  die "Usage: $0 {init|tokengen|start|stop|restart|status|repair|compact|run}"
+  die "Usage: $0 {init|tokengen|start|stop|restart|status|repair|compact|leveldbinit|run}"
```

Comment:
Should we rename `repair` and `compact` into `leveldbrepair` and `leveldbcompact`?

Reply:
done